applications/operators/olm-creating-policy.adoc

```diff
@@ -0,0 +1,39 @@
+[id="olm-creating-policy"]
+= Creating policy for Operator installations and upgrades
+include::modules/common-attributes.adoc[]
+:context: olm-creating-policy
+
+toc::[]
+
+Operators can require wide privileges to run, and the required privileges can
+change between versions. Operator Lifecycle Manager (OLM) runs with
+`cluster-admin` privileges. By default, Operator authors can specify any set of
+permissions in the ClusterServiceVersion (CSV) and OLM will consequently grant
+it to the Operator.
+
+Cluster administrators should take measures to ensure that an Operator cannot
+achieve cluster-scoped privileges and that users cannot escalate privileges
+using OLM. One method for locking this down requires cluster administrators
+auditing Operators before they are added to the cluster. Starting in
+{product-title} 4.2, however, administrators are provided tools for determining
+and constraining which actions are allowed during an Operator installation or
+upgrade.
+
+By associating an OperatorGroup with a service account with a set of privileges
+granted to it, cluster administrators can set policy on Operators to ensure they
+operate only within predetermined boundaries using RBAC rules. The Operator will
+not be able to do anything that is not explicitly permitted by those rules.
+
+This self-sufficient, limited scope installation of Operators by non-cluster
+administrators means that more of the Operator Framework tools can safely be
+made available to more users, providing a richer experience for building
+applications with Operators.
+
+include::modules/olm-policy-understanding.adoc[leveloffset=+1]
+include::modules/olm-policy-scenarios.adoc[leveloffset=+2]
+include::modules/olm-policy-workflow.adoc[leveloffset=+2]
+include::modules/olm-policy-fine-grained-permissions.adoc[leveloffset=+2]
```

Comment:
In order to have a better learning curve, modules/olm-policy-scoping-operator-install.adoc could be added before this section since it has all the examples of scoped operator install with `grant all` role.